bliss/simulator/decoder.py

```diff
@@ -11,14 +11,19 @@
 
 class ImageDecoder(nn.Module):
     def __init__(
-        self, psf, bands: Tuple[int, ...], nmgy_to_nelec_dict: dict, pixel_shift: int, ref_band: int
+        self,
+        psf,
+        bands: Tuple[int, ...],
+        physical_to_nelec_dict: dict,
```

Comment:
People may not understand the name `physical_to_nelec_dict`. Maybe `flux_calibration` would be more widely understood.

Reply:
Done.